package.json

```diff
@@ -71,6 +73,10 @@
     "node": ">= 6.0.0"
   },
   "lint-staged": {
-    "*.{js,json,md}": ["prettier --write", "git add"]
+    "*.{js,json,md}": [
+      "prettier --write",
+      "eslint --fix",
```

Comment:
Running `eslint` on `md` files?

Reply:
Yeah, it should instead do a `npm run lint` before `lint-staged` in the precommit hook.

Reply:
woops. Thanks. I'll change that